packages/graph-explorer-proxy-server/node-server.js

```diff
@@ -233,8 +239,9 @@ async function fetchData(res, next, url, options, isIamEnabled, region) {
 
     const isIamEnabled = !!req.headers["aws-neptune-region"];
     const region = isIamEnabled ? req.headers["aws-neptune-region"] : "";
+    const serviceType = isIamEnabled ? (req.headers["service-type"] ?? DEFAULT_SERVICE_TYPE) : "";
 
-    fetchData(res, next, rawUrl, requestOptions, isIamEnabled, region);
+    fetchData(res, next, rawUrl, requestOptions, isIamEnabled, region, serviceType);
   });
 
   // GET endpoint to retrieve PostgreSQL statistics summary.
```

Comment:
We need to change the Summary API calls made during connection sync to use a different API endpoint when the service type is `neptune-graph`. The base `/statistics` endpoint has been removed in recent Neptune Analytics engine versions, which are not backwards compatible. 

This is the old Summary endpoint we currently use:
```
/pg/statistics/summary
```

The new endpoint should just be:
```
/summary
```

Code pointers:
- https://github.com/Bit-Quill/graph-explorer/blob/neptune-graph/packages/graph-explorer-proxy-server/node-server.js#L248-L249
- https://github.com/Bit-Quill/graph-explorer/blob/neptune-graph/packages/graph-explorer/src/connector/openCypher/useOpenCypher.ts#L35

Reply:
fixed in https://github.com/aws/graph-explorer/pull/241/commits/4dc9e6982fe3162553882f8e18656d464606ccde